src/Raven.Studio/wwwroot/App/views/database/tasks/importDatabaseFromFile.html

```diff
@@ -180,33 +180,25 @@ <h3>Advanced</h3>
                         <div class="btn-group">
-                            <button data-bind="click: _.partial($root.copyCommandToClipboard, $root.importCommandPowerShell())" class="btn btn-default">
-                                <i class="icon-copy"></i>
-                                <span>Copy command as PowerShell</span>
-                            </button>
-                            <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown">
-                                <span class="caret"></span>
-                                <span class="sr-only">Toggle Dropdown</span>
+                            <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+                                <span data-bind="text: effectiveCommandLabel"></span>
+                                <span class="caret margin-left margin-left-xs"></span>
                             </button>
                             <ul class="dropdown-menu">
-                                <li>
-                                    <a href="#" data-bind="click: _.partial($root.copyCommandToClipboard, $root.importCommandCmd())">
-                                        <i class="icon-copy"></i> Copy as cURL (cmd)
-                                    </a>
-                                </li>
-                                <li>
-                                    <a href="#" data-bind="click: _.partial($root.copyCommandToClipboard, $root.importCommandBash())">
-                                        <i class="icon-copy"></i> Copy as cURL (bash)
-                                    </a>
-                                </li>
+                                <li><a href="#" data-bind="click: effectiveCommandType.bind(effectiveCommandType, 'PowerShell')"><span data-bind="text: getCommandTypeLabel('PowerShell')"></span></a></li>
+                                <li><a href="#" data-bind="click: effectiveCommandType.bind(effectiveCommandType, 'Cmd')"><span data-bind="text: getCommandTypeLabel('Cmd')"></span></a></li>
+                                <li><a href="#" data-bind="click: effectiveCommandType.bind(effectiveCommandType, 'Bash')"><span data-bind="text: getCommandTypeLabel('Bash')"></span></a></li>
                             </ul>
                         </div>
                         <div class="flex-grow">
-                            <input onClick="this.select();" class="form-control" readonly data-bind="value: $root.importCommandPowerShell" />
+                            <input onClick="this.select();" class="form-control" readonly data-bind="value: effectiveCommand" />
                         </div>
-                    </div>    
+                        <button class="btn btn-default" data-bind="click: copyCommandToClipboard" title="Copy import command to clipboard">
+                            <i class="icon-copy-to-clipboard"></i><span>Copy</span>
```

Comment:
I wonder if we need icon + text, or only icon?

Reply:
Using text only

Reply:
why not icon only? Like we do in other places or like github does?

Reply:
ok - changed to icon only